### ISSUE TITLE
Data is not correct

### ISSUE THREAD

**Issue by zeeshanjamal16** [Mittal-Analytics/Screener.in#118]
Quarter and Annual data fro Net Profit is not correct for Lypsa Gems. I have cross checked with BSE data and MoneyControl as well. Also can you please inform for which company current data is being displayed?

**Comment by pratyushmittal** [comment]
Hi @zeeshanjamal16 , you might be comparing Consolidated numbers shown on Screener vs Standalone numbers shown on other websites.

**Comment by zeeshanjamal16** [comment]
Yes @pratyushmittal . Thank you for clarifying. That was the exact case. Actually I was clicking on standalone but the results were displayed very late. Thats why I was confused. After your reply, I checked again and waited for around 4-5 seconds and the result was displayed. I checked the same by cloning the repo and testing on localhost. Results were fast in that case. Thanks for clarification again.
